content/docs/2.5/scalers/postgresql.md

```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
 
 **Password Authentication:**
 
-- `password` - Password for configured user to login to postgreSQL database
-variables.
+- `host:` Service URL to postgresql. Note that you should use a full svc URL as KEDA will need to contact postgresql from a different namespace
```

Comment:
```suggestion
- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace
```

Reply:
Honestly, I have copied and pasted from the current "metadata definitions", so I will change both :)